includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1665,26 +1665,33 @@ protected function build_display_items( $itemized_display_items = false ) {
 			define( 'WOOCOMMERCE_CART', true );
 		}
 
-		$items     = [];
-		$subtotal  = 0;
-		$discounts = 0;
+		$items         = [];
+		$lines         = [];
+		$subtotal      = 0;
+		$discounts     = 0;
+		$display_items = ! apply_filters( 'wc_stripe_payment_request_hide_itemization', true ) || $itemized_display_items;
 
-		// Default show only subtotal instead of itemization.
-		if ( ! apply_filters( 'wc_stripe_payment_request_hide_itemization', true ) || $itemized_display_items ) {
-			foreach ( WC()->cart->get_cart() as $cart_item_key => $cart_item ) {
-				$amount         = $cart_item['line_subtotal'];
-				$subtotal      += $cart_item['line_subtotal'];
```

Comment:
Interesting that the subtotal was already being calculated but never used.

Reply:
I tracked down this line and it was created like 4+ years ago. It was never used.

Reply:
Same 😄